systori/apps/main/management/commands/generate_swagger_file.py

```diff
@@ -0,0 +1,13 @@
+from django.core.management.base import BaseCommand, CommandError
+from drf_yasg.management.commands import generate_swagger
+
+# TODO Can't override `generate_swagger` unless `main` app is moved before `drf_yasg` in `INSTALLED_APPS`
+# See: https://docs.djangoproject.com/en/2.2/howto/custom-management-commands/#overriding-commands
+class Command(generate_swagger.Command):
+    help = "Generate swagger schema in schema.yaml"
+
+    def handle(self, *args, **options):
+        options["output_file"] = "/app/schema.yaml"
+        options["overwrite"] = True
+        options["api_url"] = "http://demo.sandbox.systori.com"
```

Comment:
does this make sense?

Reply:
What are you referring to?

If it's about the URl, I think it's okay to use the sandbox url, because that's a good default option for local development/testing. In the flutter client we change the URL anyway depending on the `dev/sandbox/production` mode